slidechain/export_test.go

```diff
@@ -59,7 +60,8 @@ func TestPegOut(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	_, err = c.DB.Exec("INSERT INTO exports (txid, amount, asset_xdr, temp, seqnum, exporter) VALUES ($1, $2, $3, $4, $5, $6)", "", amount, lumenXDR, temp, seqnum, kp.Address())
+	var anchor, pubkey [32]byte // anchor and pubkey do not matter to test this functionality
```

Comment:
Style nit: I would just declare a single thing called `zero32` and pass it for both anchor and pubkey below.